runtime-tools-quarkus-extension-parent/runtime/src/main/java/org/kie/kogito/runtime/tools/quarkus/extension/runtime/forms/FormInfo.java

```diff
@@ -13,30 +13,78 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.kie.kogito.runtime.tools.quarkus.extension.runtime.forms;
 
-import java.util.Date;
+import java.io.Serializable;
+
+public class FormInfo implements Serializable {
+
+    private final String formName;
```

Comment:
FormInfo has name, type and lastUpdated we don't need formBaseInfo for config/html/tsx